composer/trainer/trainer.py

```diff
@@ -304,6 +305,31 @@ def _is_tpu_installed() -> bool:
     import torch_xla.distributed.parallel_loader as pl
 
 
+def _maybe_create_object_store_from_uri(uri: str) -> Optional[ObjectStore]:
+    backend, bucket_name, _ = _parse_uri(uri)
+    if backend == '':
+        return None
+    if backend == 's3':
+        return S3ObjectStore(
+            bucket=bucket_name)  #bucket_uri=f'{backend}://{bucket_name}', backend_kwargs={'bucket': bucket_name})
+
+    elif backend == 'wandb':
+        raise NotImplementedError(f'There is no implementation for WandB load_object_store via URI. Please use '
+                                  'WandBLogger with log_artifacts set to True')
```

Comment:
For the load object store, wandb doesn't need `log_artifacts` set to true, right?

Reply:
fixed